softwareconfiguration/mlplan/src/main/java/ai/libs/mlpipeline_evaluation/PerformanceDBAdapter.java

```diff
@@ -152,23 +139,22 @@ public void store(ComponentInstance composition, ReproducibleInstances reproduci
 			md.update(className.getBytes());
 			byte[] digest = md.digest();
 			String hexHash = (new HexBinaryAdapter()).marshal(digest);
-			ResultSet rs = sqlAdapter.getResultsOfQuery(
-					"SELECT score FROM " + this.performanceSampleTableName + " WHERE hash_value = '" + hexHash + "'");
-			if (rs.next())
+			ResultSet rs = this.sqlAdapter.getResultsOfQuery("SELECT score FROM " + this.performanceSampleTableName + " WHERE hash_value = '" + hexHash + "'");
+			if (rs.next()) {
 				return;
+			}
 			Map<String, String> valueMap = new HashMap<>();
 			valueMap.put("composition", compositionString);
 			valueMap.put("train_trajectory", trainTrajectoryString);
 			valueMap.put("test_trajectory", testTrajectoryString);
 			valueMap.put("loss_function", className);
-			valueMap.put("evaluation_date",
-					new SimpleDateFormat("yyyy-MM-dd HH:mm:ss").format(Date.from(Instant.now())));
+			valueMap.put("evaluation_date", new SimpleDateFormat("yyyy-MM-dd HH:mm:ss").format(Date.from(Instant.now())));
 			valueMap.put("evaluation_time_ms", Long.toString(evaluationTime));
 			valueMap.put("hash_value", hexHash);
 			valueMap.put("score", Double.toString(score));
 			this.sqlAdapter.insert(this.performanceSampleTableName, valueMap);
 		} catch (JsonProcessingException | NoSuchAlgorithmException | SQLException e) {
-			e.printStackTrace();
+			logger.info("Error while storing results: {}", e);
```

Comment:
Shouldn't this be at least "warn"-level?

Reply:
true